### ISSUE TITLE
After entering the title the game touchpad doesn't work

### ISSUE THREAD

**Issue by routhing** [PojavLauncherTeam/PojavLauncher#2937]
### Describe the bug

Touch the Game-Touchpad before the MOJANG logo starts 1~3 seconds, after that when you enter the title, The game touchpad is doesn't work.

触碰 触摸板 在mojang logo出现 前 1~3秒，进入 到 标题画面 后，APP 触摸板 失灵。

Test version 1.17.1;
Pojavlauncher version: Versions in recent months have had this problem.

APP版本：1.17.1；
Pojav启动器 版本：最近几个月的版本都有这样的问题。

### The log file and images/videos

*

### Steps To Reproduce

```markdown
*
```

### Expected Behavior

I hope the author can fix this bug, because every time this happen be need to reboot the game(Force stop).😂

希望作者能修复这个问题，每次遇到这样的问题都需要重新启动游戏（强制关闭）。😂

### Platform

```markdown
- Device model: Redmi(红米) K30 5g
- CPU architecture: sdm(骁龙）765g/sm7250
- Android version: 10/MIUI12.0.7
- PojavLauncher version: Is a version released in recent months（最近几个月发布的版本）
```

**Comment by Mathias-Boulay** [comment]
known, will be fixed by #2900

**Comment by Mathias-Boulay** [comment]
Fixed on the release candidate build.
